test/models/AppModel/test.js

```diff
@@ -0,0 +1,142 @@
+    it('Should check that the error exists', async () => {
+      await dropDatabase();
+      result = await AppModel.getAll();
+      expect(result.error).is.exist;
+    });
+    it('Should return error message', async () => {
+      await dropDatabase();
+      result = await AppModel.getAll();
+      expect(result.error.message).to.eq('App not found!');
+    });
+  });
+  describe('On aggregate', async () => {
+    let name, admins, blacklists, supportedHashtags, supportedObjects, app, aggResult, result;
+    beforeEach(async () => {
+      name = faker.name.firstName();
+      admins = [faker.name.firstName().toLowerCase()];
+      blacklists = [faker.name.firstName().toLowerCase()];
+      supportedHashtags = [faker.random.string(5)];
+      supportedObjects = [faker.random.string(5)];
+      app = await AppFactory.Create({
+        name, admins, blacklists, supportedHashtags, supportedObjects,
+      });
+    });
+    it('Should return from our request: name and admins', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, admins: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(_.pick(app, ['name', 'admins']));
+    });
+    it('Should return from our request: name and blacklist users', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, blacklists: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(_.pick(app, ['name', 'blacklists']));
+    });
+    it('Should return from our request: name, blacklist users, supported Hashtags, supported Objects', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, blacklists: 1, supported_hashtags: 1, supported_objects: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.deep.eq(_.pick(app, ['name', 'blacklists', 'supported_hashtags', 'supported_objects']));
+    });
+    it('There is no blacklist as a result of our request', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { name: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, admins: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.not.have.property('blacklists');
+    });
+    it('Should return undefined, when aggregate has not result', async () => {
+      aggResult = await AppModel.aggregate([
+        { $match: { surname: app.name } },
+        {
+          $project: {
+            _id: 0, name: 1, admins: 1,
+          },
+        },
+      ]);
+      [result] = aggResult.result;
+      expect(result).to.be.undefined;
+    });
+  });
```

Comment:
in fact, you aggregate the same record in each test, and just use the project to change the display ... There should be one test to verify the project, aggregation is a very extensive method, you can check its other capabilities, read about aggregations, in aggregation, you can sort, group, and much more, you do not need to write a bunch of tests for the same operation, in fact they are useless